### ISSUE TITLE
Add images

### ISSUE THREAD

**Issue by itispx** [gomutex/godocx#14]
I saw by the commits that there is possible a way to add an image to a document. However I did not find any examples nor did I manage to figure out how to do it by myself.

Can you tell me if that is possible and if so please provide an example?

Thank you, this library is really useful.

**Comment by gomutex** [comment]
Hi,

The detailed examples are in the separate repository (You can refer pic folder inside this example repository)
https://github.com/gomutex/godocx-examples


```go
        // Example
	docx, err := godocx.NewDocument()
	if err != nil {
		log.Fatal(err)
	}

	docx.AddPicture("gopher.png", units.Inch(2.9), units.Inch(2.9))

	err = docx.SaveTo("pic.docx")
	if err != nil {
		log.Fatal(err)
	}

```

**Comment by itispx** [comment]
Was the pic example added recently? I could have sworn that I look a few days prior and did not see it.

Thank you so much!

**Comment by itispx** [comment]
I tried using this code and its not working

```go
func main() {
	docx, err := godocx.NewDocument()
	if err != nil {
		panic(err.Error())
	}

	_, err = docx.AddPicture("pfp-3.png", units.Inch(2.9), units.Inch(2.9))
	if err != nil {
		panic(err.Error())
	}

	err = docx.SaveTo("word.docx")
	if err != nil {
		panic(err.Error())
	}
}
```
Generated document:
[word.docx](https://github.com/user-attachments/files/15904829/word.docx)

**Comment by gomutex** [comment]
can you elaborate what was not working and any error you faced. Because i can see the picture in your attached word document.  Btw, it is still in beta, i am adding more elements and yet to add more.

**Comment by itispx** [comment]
I see this error when opening on desktop
![image](https://github.com/gomutex/godocx/assets/44107896/b1317b91-de03-4fdc-ab6d-c66031aa6eef)

Print from Word desktop app
![image](https://github.com/gomutex/godocx/assets/44107896/6c49948c-e4f5-4f93-95df-46b1cbb282e2)

Print from Word web app
![image](https://github.com/gomutex/godocx/assets/44107896/c359b882-552e-47b1-9c55-2ad7b5201aea)

**Comment by gomutex** [comment]
I've made few changes as per the ooxml. Please check  with `v0.1.2-beta.2` or later version.

**Comment by gomutex** [comment]
Fixed in v0.1.3-beta.1 or later versions

**Comment by itispx** [comment]
Just tested and its working fine!

Thank you so much!!!
